recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,6 @@
+cuda_compiler_version:                    # [linux64]
+  - None                                  # [linux64]
+docker_image:                             # [linux64]
+  - condaforge/linux-anvil-cos7-x86_64    # [linux64]
+cdt_name:                                 # [linux64]
+  - cos7                                  # [linux64]
```

Comment:
The cos7 docker image is default now so you can delete this file.

Reply:
Thanks for the review! Okay I have removed the file.